src/API/Site/Controllers/Assets/AssetSuggestionsController.php

```diff
@@ -0,0 +1,130 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\Assets;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\Ads\AssetSuggestionsService;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\BaseController;
+use Automattic\WooCommerce\GoogleListingsAndAds\API\TransportMethods;
+use Automattic\WooCommerce\GoogleListingsAndAds\Proxies\RESTServer;
+use WP_REST_Request as Request;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class AssetSuggestionsController
+ *
+ * @since x.x.x
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Site\Controllers\Assets
+ */
+class AssetSuggestionsController extends BaseController {
+
+	/**
+	 * Service used to populate ads suggestions data.
+	 *
+	 * @var AssetSuggestionsService
+	 */
+	protected $asset_group;
+
+	/**
+	 * AssetSuggestionsController constructor.
+	 *
+	 * @param RESTServer              $server
+	 * @param AssetSuggestionsService $asset_suggestion
+	 */
+	public function __construct( RESTServer $server, AssetSuggestionsService $asset_suggestion ) {
+		parent::__construct( $server );
+		$this->asset_group = $asset_suggestion;
+	}
+
+	/**
+	 * Register rest routes with WordPress.
+	 */
+	public function register_routes(): void {
+		$this->register_route(
+			'assets/page/suggestions',
```

Comment:
In line with my previous comment here https://github.com/woocommerce/google-listings-and-ads/pull/1784/files#r1027503068. This seems not accurate for what we got. 

I would name it `assets/suggestions`

Reply:
I have updated it to `assets/final-url/suggestions` as `assets/suggestions` is too generic and can be interpreted as suggesting assets instead of final URLs.

Reply:
Thanks for this tweak.

It's going to be any other suggestion endpoints or other `final-url` related endpoints? I like more `final-url` than `pages` for sure...  but having a "directory" `/final-url` made me think that it was going to be more endpoints here (as  children of `/final-url/`) 

If it's not the case, maybe worth it to consider just `assets/final-url-suggestions` or just `assets/url-suggestions` (I'm unsure what "final" means here)

Reply:
> It's going to be any other suggestion endpoints or other final-url related endpoints? I like more final-url than pages for sure... but having a "directory" /final-url made me think that it was going to be more endpoints here (as children of /final-url/)

For now, the other endpoint will be `assets/suggestions` and the response will include the title, descriptions, and images which can be used for creating ads.

I have seen that we have similar patterns in other controllers see here:

https://github.com/woocommerce/google-listings-and-ads/blob/19c800c34619f4c767bc11024ccac4cf3bf71393/src/API/Site/Controllers/MerchantCenter/ShippingRateSuggestionsController.php#L32

https://github.com/woocommerce/google-listings-and-ads/blob/19c800c34619f4c767bc11024ccac4cf3bf71393/src/API/Site/Controllers/MerchantCenter/TargetAudienceController.php#L93

> I'm unsure what "final" means here 

It is because the field in the Ads API and in the Ads interface is called Final URL. See here more information https://support.google.com/google-ads/answer/6080568?hl=en